cli/build.gradle

```diff
@@ -98,8 +98,8 @@ images {
         man = 'cli/resources/man'
         bundles = ['zip', 'tar.gz']
         jdk {
-            url = 'https://download.java.net/java/GA/jdk12/GPL/openjdk-12_osx-x64_bin.tar.gz'
-            sha256 = '52164a04db4d3fdfe128cfc7b868bc4dae52d969f03d53ae9d4239fe783e1a3a'
+            url = project.jdkUrl
+            sha256 = project.jdkSha256
```

Comment:
This is unfortunately not how the `jdk` property of the `images` target is meant to be used.

The JDK fully supports cross-platform linking, something we make use of in Skara. This means if you build the `:cli:images` target on a Linux x64 machine then you will actually get fully working Skara distributions for Windows, macOS *and* Linux (all x64), since we are downloading the JDKs for macOS, Windows and Linux. With this patch we only get Linux x64 distributions.

Fortunately the JDK not only supports cross-jlink across operating system, it also supports cross-linking across CPU architectures. So I would prefer to be able to write:

```
linux_x64 {
    modules = ext.modules
    launchers = ext.launchers
    man = 'cli/resources/man'
    bundles = ['zip', 'tar.gz']
    jdk {
        url = 'https://download.java.net/java/GA/jdk12/GPL/openjdk-12_linux-x64_bin.tar.gz'
        sha256 = 'b43bc15f4934f6d321170419f2c24451486bc848a2179af5e49d10721438dd56'
    }
}

linux_aarch64 {
    modules = ext.modules
    launchers = ext.launchers
    man = 'cli/resources/man'
    bundles = ['zip', 'tar.gz']
    jdk {
        url = 'https://github.com/AdoptOpenJDK/openjdk12-binaries/releases/download/jdk-12.0.2%2B10/OpenJDK12U-jdk_aarch64_linux_hotspot_12.0.2_10.tar.gz'
        sha256 = ''855f046afc5a5230ad6da45a5c811194267acd1748f16b648bfe5710703fe8c'
    }
}
```

This way we will always produce Linux AArch64 images on Linux x64 hosts. To get this to work you need to do some minor changes to `ImagesPlugin.java` (and you might have to do some adjustments to `LinkTask.java`). Do you want to take a stab at that? Or do you want me to try and hack something together?

Reply:
Ah ok, I see that now. I'm happy to have a go at redoing it.